common/src/test/java/org/apache/sedona/common/FunctionsTest.java

```diff
@@ -238,4 +235,98 @@ public void splitHeterogeneousGeometryCollection() {
 
         assertNull(actualResult);
     }
+
+    @Test
```

Comment:
Please add more tests for different types (7 types in total)

Reply:
added test case for all 7 types